### DISCUSSION TITLE
--require_secure_transport=ON

### DISCUSSION THREAD

**Discussion by m1kesk** [amacneil/dbmate#579] [Support] [answered]
I am trying to run dbmate to run migrations on mysql database with require_secure_transport=ON

I have environment variable `DATABASE_URL`
mysql://user:pass@host:3306/database_name?ssl-ca=/mysql.crt.pem

I run dbmate:
dbmate --no-dump-schema --migrations-dir "/migrations" up

however, I still got error:

Error 3159 (HY000): Connections using insecure transport are prohibited while --require_secure_transport=ON.

any suggestions, or am I correctly assuming `ssl-ca` is the option in database url ?

**Comment by dossy** [comment, marked as answer]
> I have environment variable `DATABASE_URL`
> mysql://user:pass@host:3306/database_name?ssl-ca=/mysql.crt.pem

Try:

`mysql://user:pass@host:3306/database_name?tls=true`

or:

`mysql://user:pass@host:3306/database_name?tls=skip-verify`

if your server-side SSL certificate is self-signed (presumably, by the CA represented in `/mysql.crt.pem`) or otherwise invalid.

If we wanted to support specifying a custom CA certificate, we would need to implement [mysql.RegisterTLSConfig](https://godoc.org/github.com/go-sql-driver/mysql#RegisterTLSConfig) in the dbmate mysql driver in some way, which currently isn't supported.

**Reply by m1kesk** [reply]
`tls=true` worked, without defined cert path
